Lib/idlelib/help.html

```diff
@@ -285,6 +285,8 @@ <h3>Shell menu (Shell window only)<a class="headerlink" href="#shell-menu-shell-
 </dd>
 <dt>Restart Shell</dt><dd><p>Restart the shell to clean the environment.</p>
 </dd>
+<dt>Clear and Restart</dt><dd><p>Restart the shell and clear the contents of the shell window.</p>
```

Comment:
This file is generated from the IDLE doc: Doc/librarary/idle.rst.  Edit latter instead.